cli/cmd/scan.go

```diff
@@ -213,3 +229,39 @@ func getLayerCommandsIfNeeded(sourceType sharedutils.SourceType, source string,
 
 	return layerCommands, nil
 }
+
+func getCisDockerBenchmarkResultsIfNeeded(sourceType sharedutils.SourceType,
+	source string, config *sharedconfig.Config,
+	needed bool) (dockle_types.AssessmentMap, error) {
+
+	if sourceType != sharedutils.IMAGE || !needed {
+		return nil, nil
+	}
+
+	assessmentMap, err := dockle_run.RunFromConfig(createDockleConfig(config, source))
+	if err != nil {
+		return nil, fmt.Errorf("failed to run dockle: %w", err)
+	}
+
+	logger.Infof("Image was scanned for CIS docker benchmark results.")
+	logger.Debugf("assessmentMap=%+v", assessmentMap)
+
+	return assessmentMap, nil
+}
+
+func createDockleConfig(config *sharedconfig.Config, source string) *dockle_config.Config {
+	var username, password string
+	if len(config.Registry.Auths) > 0 {
+		username = config.Registry.Auths[0].Username
+		password = config.Registry.Auths[0].Password
+	}
+	return &dockle_config.Config{
+		Debug:     log.GetLevel() == log.DebugLevel,
+		Timeout:   60 * time.Second,
```

Comment:
I think that we should make it configurable. Also, the default is pretty low, in SCN it's 5 minutes

Reply:
ack